src/features/ScreenLock/ScreenLockProvider.tsx

```diff
@@ -112,7 +112,7 @@ class ScreenLockProvider extends Component<
         this.props.history.replace(ROUTE_PATHS.DASHBOARD.path);
       });
       this.resetInactivityTimer();
-      document.title = translateRaw('SCREEN_LOCK_TAB_TITLE');
+      document.title = translateRaw('DEFAULT_PAGE_TITLE');
```

Comment:
Since we route to dashboard afterwards we should be able to remove these and let the dashboard handle the change.